kyo-core/shared/src/main/scala/kyo/internal/Param.scala

```diff
@@ -1,6 +1,7 @@
 package kyo.internal
 
 import scala.collection.mutable.LinkedHashMap
+import scala.language.implicitConversions
```

Comment:
This case would be harder to support without implicit conversions. I think it's ok to keep it as is for now.